sdk/core/azure_core_amqp/Cargo.toml

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Microsoft Corp. All Rights Reserved.
+
+# AMQP Stack for consumption by packages in the Azure SDK.
+[package]
+name = "azure_core_amqp"
+version = "0.19.0"
+description = "Rust client Azure SDK"
+readme = "README.md"
+authors = ["Microsoft Corp."]
+license = "MIT"
+repository = "https://github.com/azure/azure-sdk-for-rust"
+homepage = "https://github.com/azure/azure-sdk-for-rust"
+documentation = "https://docs.rs/azure_core_amqp"
+
+keywords = ["sdk", "azure", "rest", "iot", "cloud"]
+categories = ["api-bindings"]
+
+edition = "2021"
+
+[dependencies]
+async-std = { workspace = true }
+azure_core = { path = "../../core/azure_core" }
+time = { workspace = true }
+tracing = { workspace = true }
+url = { workspace = true }
+uuid = { workspace = true }
+fe2o3-amqp = { version = "0.11", features = [
+  "native-tls",
+  "tracing",
+  "uuid",
+], optional = true }
```

Comment:
All dependencies need to be centrally managed in the root `Cargo.toml`'s `[workspace.dependencies]` section. This is or will be validated.

Reply:
I was wondering about that - if we should include optional dependencies for just one package in the global set of packages in cargo.toml.

Reply:
Yeah, just to keep everything in one place. .NET et. al. do it as well.